### PR TITLE
NXP-29344: explorer release script

### PR DESCRIPTION
**Pull request by atchertchian**

### CODE REVIEW

File: docker/nuxeo-explorer-docker/skaffold.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ build:
         buildContext:
           localDir: {}
         buildArgs:
-          BASE_IMAGE: ${DOCKER_REGISTRY}/${ORG}/nuxeo:11.x
+          BASE_IMAGE: ${DOCKER_REGISTRY}/${ORG}/nuxeo:${NUXEO_IMAGE_VERSION}
```
Comment (atchertchian) [inline docker/nuxeo-explorer-docker/skaffold.yaml:21]: level=fatal msg="creating runner: parsing skaffold config: parsing api version: yaml: line 21: mapping values are not allowed in this context" (?)

Comment (atchertchian) [reply docker/nuxeo-explorer-docker/skaffold.yaml:21]: fixed by https://github.com/nuxeo/nuxeo-explorer/commit/5d44906f9668372d5f9e64ce10fafa63a7ef2ab2

**Comment by atchertchian** [comment]
https://jenkins.platform.dev.nuxeo.com/job/nuxeo/job/release-nuxeo-explorer/17/ went through in dry-run: will see if i can launch an actual release after review, on a work branch, before merging (kept the commit downgrading the current version with that purpose in mind for now)

**Review by ataillefer — APPROVED** [review]

**Comment by atchertchian** [comment]
Rebased to clean up commit history after test releases have been concluded successfully.
Kept last commits separated to help with review of latest changes.

**Review by ataillefer — APPROVED** [review]

**Comment by atchertchian** [comment]
Some adjustments may be needed for automated exports (https://jira.nuxeo.com/browse/NXP-29371), but let's resolve as is for this part.

**Pull request merged**